WordPress/Classes/ViewRelated/Aztec/ViewControllers/SelectPostViewController.swift

```diff
@@ -82,12 +94,14 @@ extension SelectPostViewController {
 
         let post = fetchController.object(at: indexPath)
         cell.textLabel?.text = post.titleForDisplay()
-        if post is Page {
-            cell.detailTextLabel?.text = NSLocalizedString("Page", comment: "Noun. Type of content being selected is a blog page")
-        } else {
-            cell.detailTextLabel?.text = NSLocalizedString("Post", comment: "Noun. Type of content being selected is a blog post")
+        if showsPostType {
+            if post is Page {
+                cell.detailTextLabel?.text = NSLocalizedString("Page", comment: "Noun. Type of content being selected is a blog page")
+            } else {
+                cell.detailTextLabel?.text = NSLocalizedString("Post", comment: "Noun. Type of content being selected is a blog post")
+            }
         }
-        if post.permaLink == selectedLink {
+        if isSelectedPost?(post) == true {
```

Comment:
Could omit the `== true` here

Reply:
The block is optional here so it needs something like `== true` or `isSelectedPost?(post) ?? false`.

Reply:
Ahh, sorry!